src/bom/bitcode.rs

```diff
@@ -174,18 +174,25 @@ pub fn bitcode_entrypoint(bitcode_options : &BitcodeOptions) -> anyhow::Result<i
     // Send a token to shut down the event collector thread
     sender.send(None)?;
     let summary = event_consumer.join().unwrap();
+    let exitmod =
```

Comment:
Is there any reason to not make this
```
let exitmod ec = ....
```

Reply:
Does Rust support that syntax?  If it was Haskell I would have, but I didn't know Rust supported that.